res/TensorFlowLiteRecipes/Net_InstanceNorm_003/test.rule

```diff
@@ -0,0 +1,8 @@
+# To check if custom op InstanceNorm is converted to circle builtin op
+
+RULE    "VERIFY_FILE_FORMAT"      $(verify_file_format) '=' 1
+
+RULE    "INSTANCE_NORM_EXIST"     $(op_count INSTANCE_NORM) '=' 1
+RULE    "NO_RESHAPE"              $(op_count RESHAPE) '=' 3
```

Comment:
maybe this is `RESHAPE_EXIST`

Reply:
there is 2 Reshape in the diagram in comment for `fuse_instance_norm` . which is correct?